lib/kafka.ts

```diff
@@ -58,3 +58,75 @@ export const waitForKafkaReady = async function (page: Page, name: string) {
     timeout: config.kafkaInstanceCreationTimeout
   });
 };
+
+export const manageAccess = async function (page: Page, id: string, desc: string) {
+  await page.getByTestId('actionManagePermissions').click();
```

Comment:
Is there any other identification that we could use on this? Jakub and David try to eliminate any data-testid usage from the whole test suite.

Reply:
replaced for:

`await page.getByText('Manage access').click();`

Reply:
@mmusil Can I resolve this conversation? Thanks